pkgs/tools/graphics/timg/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, git
+, pkg-config
+, graphicsmagick
+, libjpeg
+, ffmpeg
+, zlib
+, libexif
+}:
+
+stdenv.mkDerivation rec {
+  pname = "timg";
+  version = "v1.4.0";
+
+  src = fetchFromGitHub{
+    owner = "hzeller";
+    repo = "timg";
+    rev = version;
+    sha256 = "10qhjfkbazncmj07y0a6cpmi7ki0l10qzpvi2zh8369yycqqxr8y";
+  };
+
+  buildInputs = [
+    graphicsmagick
+    ffmpeg
+    libexif
+    libjpeg
+    zlib
+  ];
+
+  nativeBuildInputs = [
+    cmake
+    git
+    pkg-config
+  ];
+
+  cmakeFlags = [
+    "-DWITH_VIDEO_DECODING=On"
+    "-DWITH_VIDEO_DEVICE=On"
+    "-DWITH_OPENSLIDE_SUPPORT=Off"  # https://openslide.org/ lib not yet in nix
+  ];
+
+  meta = with lib; {
+    homepage = "https://timg.sh/";
+    description = "A terminal image and video viewer";
+    license = licenses.gpl2;
```

Comment:
Please clarify if it's `gpl2Only` or `gpl2Plus`.

Reply:
Done (`gpl2Only`).